src/screens/playlist/playlist.css

```diff
@@ -0,0 +1,25 @@
+/** @format */
+
+.playlist-wrapper {
+  margin: 2rem;
+}
+.playlist-card-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  gap: 1rem;
+  margin-top: 2rem;
+}
+.create-playlist-modal {
+  height: 200px;
```

Comment:
suggestion: try to use rem so that it will help when you are adding responsiveness...